### PR TITLE
Improve documentation for some specific methods

### PR DESCRIPTION
**Pull request by sofianehaddad**
This PR fixes some methods with missing/incorrect docstring 
It fixes also the usage of ResourceMap in GeneralizedLinearModelAlgorithm class

### CODE REVIEW

**Comment by dbarbier** [comment]
+1

File: lib/src/Uncertainty/Algorithm/MetaModel/Kriging/GeneralizedLinearModelAlgorithm.cxx [hunk 1]
```diff
@@ -879,7 +879,7 @@ void GeneralizedLinearModelAlgorithm::initializeMethod()
     method_ = 1;
 }
 
-/** Method accessor (lapack/hmat) - Protected but freind with KrigingAlgo class */
+/** Method accessor (lapack/hmat) - Protected but freind with GLMAlgo class */
```
Comment (dbarbier) [inline lib/src/Uncertainty/Algorithm/MetaModel/Kriging/GeneralizedLinearModelAlgorithm.cxx]: While you are at it, you could s/freind/friend/

**Comment by jschueller** [comment]
+1

**Comment by regislebrun** [comment]
+1

**Pull request merged**